### PR TITLE
[video] Add isLive property for Android, iOS and Web.

### PR DESCRIPTION
**Pull request by JustJoostNL**
# Why
Expo-video doesn't have any way to check if a video is a livestream.

# How
I added the isLive property on all platforms. For Web and iOS it checks if the duration is infinite, which means the video is a live stream. For Android it uses `isCurrentMediaItemLive`.

# Test Plan
Tested using bare-expo.

# Checklist

- [x] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [x] Conforms with the [Documentation Writing Style Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)
- [x] This diff will work correctly for `npx expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

File: packages/expo-video/android/src/main/java/expo/modules/video/VideoPlayer.kt [hunk 1]
```diff
@@ -137,6 +139,7 @@ class VideoPlayer(context: Context, appContext: AppContext, source: VideoSource?
       }
       if (playbackState == Player.STATE_READY) {
```
Comment (aleqsio) [inline packages/expo-video/android/src/main/java/expo/modules/video/VideoPlayer.kt:140]: shouldn't this have some sort of undefined state while the media item is loading? Are we making it explicit that you should not check this while the state is not ready?

**Review by aleqsio — APPROVED** [review]
LGTM code wise, @behenate do you think it makes sense in the package?

File: packages/expo-video/android/src/main/java/expo/modules/video/VideoPlayer.kt (hunk 1 above)
Comment (behenate) [reply packages/expo-video/android/src/main/java/expo/modules/video/VideoPlayer.kt:140]: @aleqsio I don't know if adding a special state while the media is loading isn't a bit overkill. I think we can just set it to false.

**Review by behenate — APPROVED** [review]
@aleqsio I think it's a good addition, especially for people who will be creating their own UI controls on the JS side

@JustJoostNL Thanks for another contribution!

**Review by tsapeta — APPROVED** [review]

**Pull request merged**